transaction/src/model/executable.rs

```diff
@@ -15,12 +16,25 @@ pub struct AuthZoneParams {
 /// Represents a validated transaction
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Executable {
-    pub transaction_hash: Hash,
-    pub instructions: Vec<Instruction>,
-    pub auth_zone_params: AuthZoneParams,
-    pub cost_unit_limit: u32,
-    pub tip_percentage: u32,
-    pub blobs: Vec<Vec<u8>>,
+    transaction_hash: Hash,
+    instructions: Vec<Instruction>,
+    auth_zone_params: AuthZoneParams,
+    cost_unit_limit: u32,
+    tip_percentage: u32,
+    blobs: Vec<Vec<u8>>,
+    intent_validation: IntentValidation,
```

Comment:
For your consideration, I would change this to a more generic construct `Vec<RuntimeValidation>` where 
```rust
pub enum RuntimeValidation {
    IntentHashValidation {
        intent_hash: Hash,
    },
    EpochValidation {
        start_epoch_inclusive: u64,
        end_epoch_exclusive: u64,
    },
    XyzAssertion {

    }
}
```

Reply:
Hmm I think they're both reasonable... 👍. Happy to try your approach :)